### PR TITLE
Feature/readme.md

### PR DESCRIPTION
**Pull request by nmemarcoding**
read me is ready with user story acceptance criteria mock up images and live link

### CODE REVIEW

**Review by stahljohnmark — APPROVED** [review]
Looks good, I found a few items we should probably update. For example, Whatsfordinner should be separated. The story should be bold for the reason why the site was created. Its kind of like a paper so its probably going to need a couple more updates from everyone before we present. Otherwise LGTM

File: README.md [hunk 1]
```diff
@@ -1,5 +1,47 @@
 # whatsfordinner
-Story
-Tammy wasn’t much of a cook, she knew her husband loved to make new and exciting meals and drinks. 
-For their anniversary she decided even if she didn’t know her way around the kitchen she could still surprise her husband by having all the ingredients needed for whatever he wanted to make. 
-She commissioned an app to be made so whatever protein or alcohol they had in the house could be searched and a list of meals, cocktails, and most importantly ingredients she could save and get to prepare for the meal.
\ No newline at end of file
+## User Story
+* AS A user
+* I WANT to get recipe for meal and drink
+* SO THAT I can preapr meal and drink with my ingredients
```
Comment (hjohnsick) [inline README.md]: spelling mistake on line 5, should be prepare.  SO THAT I can prepare a meal

File: README.md [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
+## Acceptance Criteria
+* GIVEN a food recipe dashboard with form inputs
+* WHEN I search for a meal by ingredients
+* THEN I am presented with  meal ingredients and recipe
+* WHEN I search for a drink by ingredients
+* THEN I am presented with  drink ingredients and recipe
+
+* WHEN I click on view recipe
+* THEN I present with pop up modal for recipe
```
Comment (hjohnsick) [inline README.md]: THEN I am presented with a pop up

File: README.md [hunk 3]
```diff
@@ -1,5 +1,47 @@
...
+* THEN I am presented with all food ingredients and  recipes
+
+* WHEN I view drink recipe
+* THEN I am presented with all drink ingredients and  recipes
+
+* THERE is a star on each meal and drink card for save meal or drink
+* WHEN I click on star
+* THEN save meal and drink to local storage
+
+* WHEN dbclick on star
```
Comment (hjohnsick) [inline README.md]: WHEN I double click

File: README.md [hunk 4]
```diff
@@ -1,5 +1,47 @@
...
+
+* WHEN I view drink recipe
+* THEN I am presented with all drink ingredients and  recipes
+
+* THERE is a star on each meal and drink card for save meal or drink
+* WHEN I click on star
+* THEN save meal and drink to local storage
+
+* WHEN dbclick on star 
+* THEN remove that food or drink from favarit meal or drink
```
Comment (hjohnsick) [inline README.md]: favorite

File: README.md [hunk 5]
```diff
@@ -1,5 +1,47 @@
...
+
+* THERE is a star on each meal and drink card for save meal or drink
+* WHEN I click on star
+* THEN save meal and drink to local storage
+
+* WHEN dbclick on star 
+* THEN remove that food or drink from favarit meal or drink 
+
+* WHEN there is a meal or drink saved in local storage
+* THEN display star for favorit meals and drinks
```
Comment (hjohnsick) [inline README.md]: favorite

File: README.md [hunk 6]
```diff
@@ -1,5 +1,47 @@
...
+
+* WHEN click on favorit meals or drinks star
```
Comment (hjohnsick) [inline README.md]: WHEN I click on favorite

File: README.md [hunk 7]
```diff
@@ -1,5 +1,47 @@
...
+* THEN display all favorits meals or drink
```
Comment (hjohnsick) [inline README.md]: favorites

**Review by hjohnsick — CHANGES_REQUESTED** [review]
There should be a section for languages/frameworks that includes JavaScript, JQuery, HTML, CSS, and Bulma.  I also don't think any of the user story info should be in the readme.  That should have been created at the start as issues of what needed to be completed.  I think it should just be a description of what the application does, the languages used, link to deployed site, and images of site.

**Review by stahljohnmark — APPROVED** [review]
As discussed we should adjust the narrative other than that. LGTM

**Review by hjohnsick — APPROVED** [review]

**Pull request merged**